InOut/CMakeLists.txt

```diff
@@ -169,24 +170,22 @@ else()
 endif()
 
 if(USE_PORTMIDI)
-    set(pmidi_LIBS ${PORTMIDI_LIBRARY})
     include_directories(${PORTMIDI_INCLUDE_PATH})
+	make_plugin(pmidi pmidi.c)
     if(LINUX)
         if(NEED_PORTTIME)
-           list(APPEND pmidi_LIBS ${PORTTIME_LIBRARY})
-           include_directories(${PORTTIME_INCLUDE_PATH})
-        endif()
-    endif()
-    if(WIN32)
-      if(NEED_PORTTIME)
-        include_directories(${PORTTIME_INCLUDE_PATH})
-      endif()
-        list(APPEND pmidi_LIBS ${CSOUND_WINDOWS_LIBRARIES})
-    endif()
-    if(LINUX AND ALSA_LIBRARY)
-        list(APPEND pmidi_LIBS ${ALSA_LIBRARY})
+			include_directories(${PORTTIME_INCLUDE_PATH})

```

Comment:
Indentation is off (using tabs instead of spaces) in some (but not all) cmake changes.